src/CalculatorUI/Controls/OperatorPanelButton.cs

```diff
@@ -0,0 +1,108 @@
+
+            public string Glyph
+            {
+                get { return (string)GetValue(GlyphProperty); }
+                set { SetValue(GlyphProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for Glyph.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty GlyphProperty =
+                DependencyProperty.Register(nameof(Glyph), typeof(string), typeof(OperatorPanelButton), new PropertyMetadata(default(string)));
+
+            public double GlyphFontSize
+            {
+                get { return (double)GetValue(GlyphFontSizeProperty); }
+                set { SetValue(GlyphFontSizeProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for GlyphFontSize.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty GlyphFontSizeProperty =
+                DependencyProperty.Register(nameof(GlyphFontSize), typeof(double), typeof(OperatorPanelButton), new PropertyMetadata(default(double)));
+
+            public double ChevronFontSize
+            {
+                get { return (double)GetValue(ChevronFontSizeProperty); }
+                set { SetValue(ChevronFontSizeProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for ChevronFontSize.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty ChevronFontSizeProperty =
+                DependencyProperty.Register(nameof(ChevronFontSize), typeof(double), typeof(OperatorPanelButton), new PropertyMetadata(default(double)));
+
+            public Flyout FlyoutMenu
+            {
+                get { return (Flyout)GetValue(FlyoutMenuProperty); }
+                set { SetValue(FlyoutMenuProperty, value); }
+            }
+
+            // Using a DependencyProperty as the backing store for FlyoutMenu.  This enables animation, styling, binding, etc...
+            public static readonly DependencyProperty FlyoutMenuProperty =
+                DependencyProperty.Register(nameof(FlyoutMenu), typeof(Flyout), typeof(OperatorPanelButton), new PropertyMetadata(default(Flyout)));
+
+            protected override void OnApplyTemplate()
+            {
+                if (FlyoutMenu != null)
+                {
+                    FlyoutMenu.Closed += FlyoutClosed;
+                }
+            }
+
+            protected override void OnToggle()
+            {
+                base.OnToggle();
+
+                // CSHARP_MIGRATION: TODO:
+                // Check when IsChecked is null
+                if (IsChecked.Value)
+                {
+                    FlyoutMenu.ShowAt(this);
+                }
+            }
+
+            // CSHARP_MIGRATION: TODO:
+            // Convert Object to object. Optimize Sharpify for case change of Object, String...
```

Comment:
Maybe we should remove this comment from the code and write down the memo into other places.